Change log level from warning/error to debug to avoid rsyslog freeze

vasant17 opened the pull request:
What I did:

Change log level from warning/error to debug.

Why I did:

Due to execessive logging we were observing that, orchagent or syncd or both get stuck in send() system call of logging. This was happening because rsyslogd was NOT reading from socket buffer. Verified this by sending SIGSTOP and SIGCONT to rsyslogd. This problem happened everytime we ran Dynamic Port Brteakout scale test case in VS container. (test_port_dpb.py)

How I verified:

Ran DPB test case again in "DPB smoketest VS " Jenkins job, the problem did NOT occur
Refer DPB smoketest job# 136 and 137

Commit:
Change log level from warning/error to debug to avoid running into rsyslog freeze due to execessive logging

zhenggen-xu commented on vslib/src/sai_vs_generic_get.cpp:
```diff
@@ -107,7 +107,7 @@ sai_status_t internal_vs_generic_get(
             if (status != SAI_STATUS_SUCCESS)
             {
-                SWSS_LOG_ERROR("%s read only not implemented on %s",
+                SWSS_LOG_DEBUG("%s read only not implemented on %s",
```
I think it is better to be INFO level, so we can change level to catch this but not mixed with DEBUG which is too verbose.

vasant17 replied:
Sure

zhenggen-xu commented on vslib/src/sai_vs_hostintf.cpp:
```diff
@@ -671,7 +671,7 @@ void hostif_info_t::process_packet_for_fdb_event(
     if (fi.fdb_entry.bv_id == SAI_NULL_OBJECT_ID)
     {
-        SWSS_LOG_WARN("skipping mac learn for %s, since BV_ID was not found for mac",
+        SWSS_LOG_DEBUG("skipping mac learn for %s, since BV_ID was not found for mac",
```
In what case we hit this?  This needs to be understood, and I don't think we should filter this out.

vasant17 replied:
We hit this during MAC learning. More precisely, when we have physical port and VLAN but fail to find bvid and bridge port ID. Also note that this happens for ports created during system initialization. For the ports created during breakout, we hit the following message:
```
 SWSS_LOG_WARN("failed to get port vlan id from port %s",
                        sai_serialize_object_id(portid).c_str());
```
I dont know how can this happen, because when we create port, we put it in default VLAN. When we try to get VLAN ID attribute of a port, it should always succeed. 

For now let me keep this log as warning and understand better.

zhenggen-xu commented:
Have you checked the log level on real HW?  We could hit the same issue if the log is also excessive on HW.

vasant17 commented:
> Have you checked the log level on real HW? We could hit the same issue if the log is also excessive on HW.

Yes I did,  
Port add ERR logs:
```
May  6 05:46:08.802589 lnos-x1-a-asw01 INFO syncd#syncd: [none] brcm_sai_create_port:2453 Create called for bcmport 70 with lanes 1 speed 25000
May  6 05:46:08.822577 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_queue_attribute:842 Unknown qos queue attribute 10 passed
May  6 05:46:08.824178 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 39 times: [ [none] brcm_sai_get_queue_attribute:842 Unknown qos queue attribute 10 passed]
May  6 05:46:08.824331 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.
May  6 05:46:08.824822 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 9 times: [ [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.]
May  6 05:46:08.824945 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_queue_attribute:842 Unknown qos queue attribute 10 passed
May  6 05:46:08.826331 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 47 times: [ [none] brcm_sai_get_queue_attribute:842 Unknown qos queue attribute 10 passed]
May  6 05:46:08.826443 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.
May  6 05:46:08.826855 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 9 times: [ [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.]
May  6 05:46:08.826979 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.827061 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.827140 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.827219 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.829534 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.829674 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.829758 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.829837 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.829915 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.829994 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.830071 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.830149 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.830227 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.830304 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.830382 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.830462 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.830556 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.830711 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 2 times: [ [none] brcm_sai_get_port_attribute:1791 Error processing port attributes]
May  6 05:46:08.830844 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 53 passed
May  6 05:46:08.830977 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.831104 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 54 passed
May  6 05:46:08.831227 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.831348 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 55 passed
May  6 05:46:08.831469 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.831586 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 56 passed
May  6 05:46:08.831711 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.831832 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 58 passed
May  6 05:46:08.831951 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.832071 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 59 passed
May  6 05:46:08.832188 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.832307 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 61 passed
May  6 05:46:08.832423 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.832551 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 62 passed
May  6 05:46:08.832702 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.832823 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 63 passed
May  6 05:46:08.832943 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.833061 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 64 passed
May  6 05:46:08.833177 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.833294 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 65 passed
May  6 05:46:08.833412 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.833529 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 66 passed
May  6 05:46:08.833647 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.833764 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 67 passed
May  6 05:46:08.833881 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.834000 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 68 passed
May  6 05:46:08.834119 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.834239 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 69 passed
May  6 05:46:08.834357 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.834475 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 70 passed
May  6 05:46:08.834592 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.834710 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 71 passed
May  6 05:46:08.834830 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.834949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 72 passed
May  6 05:46:08.835065 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.835182 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 73 passed
May  6 05:46:08.835299 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.835415 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 79 passed
May  6 05:46:08.835532 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.835650 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 84 passed
May  6 05:46:08.835768 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.835885 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 85 passed
May  6 05:46:08.836001 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.836119 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 87 passed
May  6 05:46:08.836235 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.836353 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 93 passed
May  6 05:46:08.836469 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.836606 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.836728 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.836847 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.836967 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.837092 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.837212 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.837330 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.837448 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.837572 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.837704 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.837825 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.837945 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.838064 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.838184 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.838306 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.838430 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.838551 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.838670 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 2 times: [ [none] brcm_sai_get_port_attribute:1791 Error processing port attributes]
May  6 05:46:08.838786 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 53 passed
May  6 05:46:08.838904 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.839022 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 54 passed
May  6 05:46:08.839139 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.839257 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 55 passed
May  6 05:46:08.839375 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.839493 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 56 passed
May  6 05:46:08.839610 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.839729 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 58 passed
May  6 05:46:08.839847 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.839966 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 59 passed
May  6 05:46:08.840082 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.840199 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 61 passed
May  6 05:46:08.840317 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.840433 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 62 passed
May  6 05:46:08.840576 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.840702 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 63 passed
May  6 05:46:08.840825 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.840943 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 64 passed
May  6 05:46:08.841063 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.841183 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 65 passed
May  6 05:46:08.841300 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.841419 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 66 passed
May  6 05:46:08.841538 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.841656 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 67 passed
May  6 05:46:08.841774 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.841894 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 68 passed
May  6 05:46:08.842016 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.842136 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 69 passed
May  6 05:46:08.842254 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.842372 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 70 passed
May  6 05:46:08.842495 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.842615 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 71 passed
May  6 05:46:08.842734 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.842852 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 72 passed
May  6 05:46:08.842971 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.843087 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 73 passed
May  6 05:46:08.843204 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.843321 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 79 passed
May  6 05:46:08.843438 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.843555 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 84 passed
May  6 05:46:08.843672 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.843790 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 85 passed
May  6 05:46:08.843878 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.843998 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 87 passed
May  6 05:46:08.844114 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.844230 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 93 passed
May  6 05:46:08.844352 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.844439 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.
May  6 05:46:08.844766 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 9 times: [ [none] brcm_sai_get_scheduler_group_attribute:497 First retreive child list from port or scheduler module.]
May  6 05:46:08.844766 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.844796 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.844822 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.844928 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846401 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846496 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846496 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846496 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846531 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846531 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846554 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846554 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 0 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_ingress_priority_group_attribute:251 Unknown ing prio group attribute 2 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: message repeated 2 times: [ [none] brcm_sai_get_port_attribute:1791 Error processing port attributes]
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 53 passed
May  6 05:46:08.846668 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846712 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 54 passed
May  6 05:46:08.846712 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 55 passed
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 56 passed
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 58 passed
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846812 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 59 passed
May  6 05:46:08.846854 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846854 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 61 passed
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 62 passed
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 63 passed
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846949 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 64 passed
May  6 05:46:08.846975 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.846996 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 65 passed
May  6 05:46:08.846996 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847020 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 66 passed
May  6 05:46:08.847036 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847052 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 67 passed
May  6 05:46:08.847052 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847052 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 68 passed
May  6 05:46:08.847091 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 69 passed
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 70 passed
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 71 passed
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 72 passed
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847220 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 73 passed
May  6 05:46:08.847252 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847252 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 79 passed
May  6 05:46:08.847281 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847281 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 84 passed
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 85 passed
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 87 passed
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1784 Unknown port attribute 93 passed
May  6 05:46:08.847393 lnos-x1-a-asw01 ERR syncd#syncd: [none] brcm_sai_get_port_attribute:1791 Error processing port attributes
May  6 05:46:08.847393 lnos-x1-a-asw01 NOTICE syncd#syncd: :- onPostPortCreate: discovered 149 new objects (including port) after creating port VID: oid:0x1000000001a42
May  6 05:46:08.847435 lnos-x1-a-asw01 NOTICE syncd#syncd: :- onPostPortCreate: putting ALL new discovered objects to redis for port oid:0x1000000001a42
May  6 05:46:08.861257 lnos-x1-a-asw01 NOTICE syncd#syncd: :- redisUpdatePortLaneMap: added 1 lanes to redis lane map for port RID oid:0x100000046
May  6 05:46:08.861257 lnos-x1-a-asw01 NOTICE syncd#syncd: :- post_port_create: post port create actions succeeded
```

Port delete ERR logs:

```
May  6 05:46:07.741375 lnos-x1-a-asw01 NOTICE syncd#syncd: :- get_port_related_objects: obtained 39 port oid:0x100000044 related RIDs
May  6 05:46:07.741375 lnos-x1-a-asw01 INFO syncd#syncd: [none] brcm_sai_remove_port:2747 Remove called for port object 0x       100000044 bcmport 68
May  6 05:46:07.752760 lnos-x1-a-asw01 NOTICE syncd#syncd: :- onPostPortRemove: removed 4 lanes from redis lane map for port RID oid:0x100000044
May  6 05:46:07.752760 lnos-x1-a-asw01 NOTICE syncd#syncd: :- post_port_remove: post port remove actions succeeded
May  6 05:46:07.755275 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c6
May  6 05:46:07.755354 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c6, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.755563 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c6
May  6 05:46:07.755634 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c6, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.755844 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c7
May  6 05:46:07.755910 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c7, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.755972 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c7
May  6 05:46:07.756033 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c7, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.756201 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c8
May  6 05:46:07.756257 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c8, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.756319 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c8
May  6 05:46:07.756447 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c8, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.756621 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c9
May  6 05:46:07.756621 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c9, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.756780 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019c9
May  6 05:46:07.756847 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019c9, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.756910 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019ca
May  6 05:46:07.756970 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019ca, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757081 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019ca
May  6 05:46:07.757108 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019ca, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757237 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cb
May  6 05:46:07.757335 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cb, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757335 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cb
May  6 05:46:07.757443 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cb, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757565 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cc
May  6 05:46:07.757661 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cc, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757661 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cc
May  6 05:46:07.757764 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cc, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.757865 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cd
May  6 05:46:07.757891 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cd, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758025 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019cd
May  6 05:46:07.758184 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019cd, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758242 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019ce
May  6 05:46:07.758339 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019ce, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758339 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019ce
May  6 05:46:07.758378 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019ce, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758529 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d0
May  6 05:46:07.758529 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d0, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758563 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d0
May  6 05:46:07.758683 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d0, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758683 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d1
May  6 05:46:07.758786 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d1, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758812 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d1
May  6 05:46:07.758923 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d1, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.758947 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d2
May  6 05:46:07.759046 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d2, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759142 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d2
May  6 05:46:07.759167 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d2, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759167 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d3
May  6 05:46:07.759291 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d3, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759291 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d3
May  6 05:46:07.759404 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d3, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759430 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d4
May  6 05:46:07.759505 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d4, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759505 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d4
May  6 05:46:07.759605 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d4, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759691 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d5
May  6 05:46:07.759691 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d5, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759793 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d5
May  6 05:46:07.759818 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d5, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.759918 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d6
May  6 05:46:07.759918 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d6, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760049 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d6
May  6 05:46:07.760077 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d6, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760193 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d7
May  6 05:46:07.760193 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d7, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760310 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d7
May  6 05:46:07.760310 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d7, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760411 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d8
May  6 05:46:07.760411 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d8, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760501 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d8
May  6 05:46:07.760606 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d8, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760687 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d9
May  6 05:46:07.760687 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d9, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760774 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x150000000019d9
May  6 05:46:07.760835 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x150000000019d9, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.760902 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019bd
May  6 05:46:07.760963 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019bd, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761030 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019be
May  6 05:46:07.761092 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019be, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761180 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019bf
May  6 05:46:07.761209 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019bf, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761290 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019c0
May  6 05:46:07.761314 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019c0, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761400 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019c1
May  6 05:46:07.761400 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019c1, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761520 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019c2
May  6 05:46:07.761520 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019c2, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761642 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019c3
May  6 05:46:07.761702 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019c3, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761763 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x1a0000000019c4
May  6 05:46:07.761828 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x1a0000000019c4, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761916 lnos-x1-a-asw01 ERR syncd#syncd: :- translate_vid_to_rid: unable to get RID for VID: 0x10000000019bb
May  6 05:46:07.761976 lnos-x1-a-asw01 WARNING syncd#syncd: :- processFlexCounterEvent: port VID oid:0x10000000019bb, was not found (probably port was removed/splitted) and will remove from counters now
May  6 05:46:07.761976 lnos-x1-a-asw01 NOTICE syncd#syncd: :- removePortDebugCounters: Trying to remove nonexisting port debug counter Ids 0x10000000019bb
```

Commit:
Addressed code-review comments
